db/schema.rb

```diff
@@ -10,11 +10,20 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_04_04_140723) do
+ActiveRecord::Schema.define(version: 2020_05_10_184356) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "answers", force: :cascade do |t|
+    t.text "answer_text"
```

Comment:
since we are already in the answers table, lets rename this as:

```suggestion
    t.text "name"
```

Reply:
lol i just found out that this is the schema :D :D :D

Reply:
Yes I am just about to say that, LOL